packages/@aws-cdk/aws-redshift-alpha/lib/table.ts

```diff
@@ -272,7 +272,7 @@ export class Table extends TableBase {
       properties: {
         tableName: {
           prefix: props.tableName ?? cdk.Names.uniqueId(this),
-          generateSuffix: !props.tableName ? 'true' : 'false',
+          generateSuffix: (props.tableName == null).toString(),
```

Comment:
To help me understand, what's the difference?

Reply:
That's fine with me, I just want to make sure if I misunderstood anything and if those two expressions have a difference.